common/listener.go

```diff
@@ -131,3 +131,12 @@ func (listener *Listener) Encode(w http.ResponseWriter, response interface{}) er
 	}
 	return err
 }
+
+// LoggerHandler Also TODO: inject environment CNS and restrict API's
+// i.e. SF can't call URI's based on which env cns is running
+func LoggerHandler(handler func(http.ResponseWriter, *http.Request)) func(http.ResponseWriter, *http.Request) {
```

Comment:
Remove for this PR, but potentially worthwhile using middleware to block handler availability based on CNS orchestrator